### DISCUSSION TITLE
Small design improvement for comments

### DISCUSSION THREAD

**Discussion by emvaized** [Docile-Alligator/Infinity-For-Reddit#1537] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

I think it would be great if Infinity provided an option for more compact view for minimized comments (those with hidden bottom actions). 

### Additional context

Currently, it shows "Edited" text for edited posts, and "n pts" for points count: 

![2023-09-08_08-19](https://github.com/Docile-Alligator/Infinity-For-Reddit/assets/37851576/49b3c2e9-0d86-456e-a33e-8329594eab8a)

In non-English locales text may become even more prominent. For example, please see russian language, where "pts" becomes much more wordy "пойнтов":

![in russian locale](https://github.com/Docile-Alligator/Infinity-For-Reddit/assets/37851576/7f5ffb88-94c3-4fc4-bbd6-29f13a0d9fa2)

### How important is this feature?

Nice to have

### Solution

I propose to show points count in a small oval bubble, without any words. And instead of "Edited" text, just show a [pen icon](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Aedit%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4024). 
Here's a quick mockup of how it may look: 

![new](https://github.com/Docile-Alligator/Infinity-For-Reddit/assets/37851576/240daf7a-4b0e-4386-a779-217f31d2a2dc)

I think it looks much more pleasant and easily readable.

**Comment by Gtt1229** [comment]
Do you use your own API or Patreon?

**Reply by emvaized** [reply]
I use my own API

**Reply by Gtt1229** [reply]
> I use my own API

This is pretty easy to adjust in your own build if you're inclined. I have my comments adjusted as such:


![Screenshot_20240218_000535_Infinity.jpg](https://github.com/Docile-Alligator/Infinity-For-Reddit/assets/23534272/81da4b4b-ace2-4534-8978-7fcb1a0c304f)




I don't care for a bubble, but I may do the pencil icon that you suggested. I'll fork something in a day or so.

**Reply by Gtt1229** [reply]
@emvaized I wanted to share why I don't like the circular points. I added the edited pencil instead of the word. I have not fixed up the points 100%, the issue is that after 3 digits the circle becomes a pretty bad oval, and the point texts will have to be shrunken to fit, which would be difficult to read.

![studio64_edw6luCWEt](https://github.com/Docile-Alligator/Infinity-For-Reddit/assets/23534272/838421e7-7b6e-4dda-be61-5cb75b83b6a0)

Is there an app that currently (or did*) implement the circular points?

**Reply by emvaized** [reply]
@Gtt1229 Hmm, I guess you're right! However, I use auto-build Google script, so probably may need some research first on how to build this app manually. 

> Is there an app that currently (or did*) implement the circular points?

I'm not sure about this. I came up with this idea to better visually distinguish between points and time digits. I guess you're right, it doesn't look very good with >3 digits. Maybe then a simple square with rounded corners would fit better as a background for points?

**Reply by Gtt1229** [reply]
@emvaized I have it looking like such now:

![image](https://github.com/Docile-Alligator/Infinity-For-Reddit/assets/23534272/c4f0869a-65a8-41c5-bdd8-3058525fed13)

When using the auto-built, you can specify a repository or you can build through Android Studio. I have to clean up my fork because I was trying some other things, but I'll let you know when it is updated. Most likely Wednesday.

**Reply by emvaized** [reply]
Okay, I see. Keep me posted! I guess I'll also look into creating my own fork. However, it will probably lead to extra-hassle to keep it up-to-date on every main repo's big update.
